getting_started.ipynb

```diff
@@ -15,7 +15,7 @@
     "To execute a given 'cell' of this notebook, place the cursor in the cell and press the 'play' icon, or simply press shift+enter together. Some cells will take longer to run, and you will see a [$\\ast$] to the left of the cell while AE is still working.\n",
     "\n",
     "## Step 0: Setup\n",
-    "These cells import our custom library *climakitae*, and any other specialized libraries needed for a given notebook."
+    "This cell imports our AE toolkit for climate analysis [climakitae](https://github.com/cal-adapt/climakitae), and any other specialized libraries needed for a given notebook."
```

Comment:
The wording here might be better written as: 
"This cell imports the python library climakitae, our AE toolkit for climate data analysis."